### ISSUE TITLE
Inconsistent naming

### ISSUE THREAD

**Issue by jgosmann** [tcstewar/ctn_benchmarks#4]
The repository is named `ctn_benchmarks` (plural), but you import `ctn_benchmark` (singular).

**Comment by tcstewar** [comment]
Yeah, I keep meaning to fix that....  but I also keep meaning to think of a different name for these things too, to reflect the fact that it's really about making experiments that can be used as benchmarks but can also be used for exploration and whatnot....  but nothing has been coming to mind for a better name.
